src/main/java/core/basesyntax/exception/RegistrationException.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.exception;
+
+public class RegistrationException extends RuntimeException {
+    // Constructors
+    public RegistrationException() {
+        super();
+    }
+
+    public RegistrationException(String message) {
+        super(message);
+    }
+
```

Comment:
```suggestion
    public RegistrationException(String message) {
        super(message);
    }
```

Reply:
not fixed prev comment: remove constructor without parameters.